src/features/workflow/WorkflowEditor.tsx

```diff
@@ -145,7 +148,11 @@ const WorkflowEditor: React.FC<WorkflowEditorProps> = ({
                     onChangeScript={(i:number, script:string) => {
                       if (dataset?.transform?.steps) {
                         dispatch(changeWorkflowTransformStep(i, script))
+                        if (run && run.id) {
+                          dispatch(removeEvent(run.id))
+                        }
```

Comment:
If we clear the run here, then we won't have the ability to display the output.

We should remove these lines, and instead call `removeEvent` in `runScript`, before we actually dispatch `applyWorkflowTransform`

Reply:
This looks like it got skipped!